NNAPI: Support tensor arguments in quantize

dreiss opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #77772
* **#77771**

I have an upcoming diff that changes quantize to pass tensor arguments
instead of Python scalars, so check for that during conversion.

Test Plan:
Unit test.

dreiss committed:
NNAPI: Support tensor arguments in quantize

I have an upcoming diff that changes quantize to pass tensor arguments
instead of Python scalars, so check for that during conversion.

Test Plan:
Unit test.

[ghstack-poisoned]

axitkhurana reviewed (approved):
This was because of recent pytorch changes?

dreiss commented:
No.  The next diff in the stack makes this necessary.

dreiss commented:
@pytorchbot merge please

dreiss committed:
rebase on "NNAPI: Support tensor arguments in quantize"


I have an upcoming diff that changes quantize to pass tensor arguments
instead of Python scalars, so check for that during conversion.

Test Plan:
Unit test.

[ghstack-poisoned]
